source/client-side-encryption/client-side-encryption.md

```diff
@@ -0,0 +1,2483 @@
+# Client Side Encryption
+
+- Status: Accepted
+- Minimum Server Version: 4.2 (CSFLE), 6.0 (Queryable Encryption)
+- Version: 1.13.0
+
+______________________________________________________________________
+
+## Abstract
+
+MongoDB 4.2 introduced support for client side encryption, guaranteeing that sensitive data can only be encrypted and
+decrypted with access to both MongoDB and a separate key management provider (supporting AWS, Azure, GCP, a local
+provider, and KMIP). Once enabled, data can be seamlessly encrypted and decrypted with minimal application code changes.
+6.0 introduced the next generation of client side encryption based on a Structured Encryption framework which allows
+expressive encrypted search operations. This spec covers both capabilities - 1st generation, "Client Side Field Level
+Encryption" and generation 2, "Queryable Encryption" - as the associated core cryptographic library and supporting
+drivers share a common codebase.
+
+## META
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## Terms
+
+encrypted MongoClient\
+A MongoClient with client side encryption enabled.
+
+data key\
+A key used to encrypt and decrypt BSON values. Data keys are encrypted with a key management service (e.g. AWS
+KMS) and stored within a document in the MongoDB key vault collection (see
+[Key vault collection schema for data keys](#key-vault-collection-schema-for-data-keys) for a description of the data
+key document). Therefore, a client needs access to both MongoDB and the external KMS service to utilize a data key.
+
+MongoDB key vault collection\
+A MongoDB collection designated to contain data keys. This can either be co-located with
+the data-bearing cluster, or in a separate external MongoDB cluster.
+
+Key Management Service (KMS)\
+An external service providing fixed-size encryption/decryption. Only data keys are
+encrypted and decrypted with KMS.
+
+KMS providers
+
+> A map of KMS providers to credentials. Configured client-side. Example:
```

Comment:
Is the blockquote intentional here?

An alternative approach to formatting the terminology lists would be to apply bold formatting to the leading terms and/or use a bulleted list. Doing both would probably be best for readability.